components/sysutils/screen/Makefile

```diff
@@ -51,7 +51,6 @@ COMPONENT_PREP_ACTION += ( cd $(@D) ; $(CONFIG_SHELL) autogen.sh );
 
 CONFIGURE_OPTIONS += --enable-colors256
 CONFIGURE_OPTIONS += --with-sys-screenrc=/etc/screenrc
-CONFIGURE_OPTIONS += --disable-socket-dir
 CONFIGURE_OPTIONS += --infodir=$(CONFIGURE_INFODIR)
```

Comment:
I think this line is a duplicate.  Could you please double check and remove if it really is?

Thank you.

Reply:
I don't understand. What exactly is this line a duplicate of?

Reply:
I'm sorry for confusion.

I think it is a duplicate of this one: https://github.com/OpenIndiana/oi-userland/blob/b57dded888d8cecb18bcb6752babd6932af0b54d/make-rules/configure.mk#L152

To confirm that please run the make and check the output.  You should see `--infodir=...` twice with the same value.  If you do, then it is a duplicate and it should be removed here.

Thank you.